osbs-client.spec

```diff
@@ -32,20 +29,17 @@
 %global osbs_obsolete_vr 0.14-2
 
 Name:           osbs-client
-Version:        0.65
-%if "x%{postrelease}" != "x0"
-Release:        %{release}.%{postrelease}.git.%{shortcommit}%{?dist}
-%else
-Release:        %{release}%{?dist}
-%endif
+Version:        0.64.1
+Release:        1%{?dist}
 
 Summary:        Python command line client for OpenShift Build Service
 Group:          Development/Tools
 License:        BSD
 URL:            https://github.com/containerbuildsystem/osbs-client
-Source0:        https://github.com/containerbuildsystem/osbs-client/archive/%{commit}/osbs-client-%{commit}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
```

Comment:
ugh. This should point to an upstream resource for your tarball. 
The goal is to separate upstream source and vendor modifications.